bindings/pyroot_experimental/PyROOT/src/TTreePyz.cxx

```diff
@@ -0,0 +1,143 @@
+
+// Bindings
+#include "CPyCppyy.h"
+#include "PyROOTStrings.h"
+#include "PyROOTPythonize.h"
+#include "CPPInstance.h"
+#include "ProxyWrappers.h"
+#include "Converters.h"
+#include "Utility.h"
+
+// ROOT
+#include "TClass.h"
+#include "TInterpreter.h"
+#include "TMethod.h"
+
+#include "TTree.h"
+#include "TBranch.h"
+#include "TBranchElement.h"
+#include "TBranchObject.h"
+#include "TLeaf.h"
+#include "TLeafElement.h"
+#include "TLeafObject.h"
+#include "TStreamerElement.h"
+#include "TStreamerInfo.h"
+
+#include "CustomPyTypes.h"
+#include <list>
+
+using namespace CPyCppyy;
+
+static inline TClass *OP2TCLASS(CPPInstance *pyobj)
```

Comment:
Can you make this function look like a function and not like a preprocessor macro? Or in other words, could you please follow ROOT's coding convention here?